src/master/develop/indexes.md

```diff
@@ -1,9 +1,11 @@
 # Secondary indexes
```

Comment:
We only describe ZSet. We should also add documentation about indexes made with simple references.